components/track/TrackItem.vue

```diff
@@ -1,6 +1,6 @@
 <script lang="ts" setup>
-import type { NuxtIconName } from ".nuxt/nuxt-icons";
-import type { RoutesNamedLocations } from ".nuxt/typed-router";
+import type { NuxtIconName } from "~/.nuxt/nuxt-icons";
```

Comment:
We should probably import from `#build/*` instead of `~/.nuxt/*` here. `#build` is an alias for the `.nuxt` directory, if I'm not mistaken

Reply:
Fixed.